portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer Dai's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
+      <script src="script.js"></script>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <div id="header">
+        <h1>Jennifer Dai's Portfolio</h1>
+        <a href="https://ca.linkedin.com/in/jennifer-dai">LinkedIn</a>
+        <a href="https://github.com/nif33">Github</a>
+      </div>
+
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomGreeting()">Tell me more!</button>
       <div id="greeting-container"></div>
+
+      <h2>Background</h2>
+      <p>I'm a 2nd year Computing Science student at Simon Fraser University. Three fun facts about me: 
+          I have a pet dog named William, my favourite place I've travelled to is Montreal, 
+          and my first day of work at Google was also my 20th birthday!
+      </p>
+
+      <h2>Photos of my Dog</h2>
+      <p>This is my dog: William</p>
+      <img class="dog-slides" src="/images/dog/dog0.jpg">
+      <img class="dog-slides" src="/images/dog/dog1.jpg">
+      <img class="dog-slides" src="/images/dog/dog2.jpg">
+
+      <h2>My Favourite Places</h2>
+      <p>I haven't travelled much, but my favourite place I've been to is Montreal</p>
+      <img class="place-slides" src="/images/place/place0.jpg">
+      <img class="place-slides" src="/images/place/place1.jpg">
+      <img class="place-slides" src="/images/place/place2.jpg">
+      
     </div>
+  <script src="script.js"></script>
```

Comment:
Similar to my question above. This time why two scripts? I think you only need one. Please correct me if I'm wrong.

Reply:
Make sure to resolve comments after you fix them.